python/ccf/clients.py

```diff
@@ -480,42 +532,56 @@ def call(self, *args, **kwargs):
                 time.sleep(0.1)
 
     def get(self, *args, **kwargs):
+        """
+        Issue ``GET`` request.
+        Wrapper for :py:meth:`ccf.clients.CCFClient.call` with ``http_verb`` set to ``GET``.
```

Comment:
I'd be tempted to just say "Issue GET request. See :py:meth:`ccf.clients.CCFClient.call`"